extensions/packageManager/src/pages/Library.tsx

```diff
@@ -78,6 +78,7 @@ const Library: React.FC = () => {
   const [readmeContent, setReadmeContent] = useState<string>('');
   const [versionOptions, setVersionOptions] = useState<IContextualMenuProps | undefined>(undefined);
   const [isUpdate, setIsUpdate] = useState<boolean>(false);
+  const [readmeHidden, setReadmeHidden] = useState<boolean>(true);
```

Comment:
This may be my personal opinion here, but I find confusing when the variable value and the thing being hidden/shown matches. (not sure how to word this) So I would suggest something like `const [showReadme] = useState(false)` instead.